src/server/game/Entities/AreaTrigger/AreaTrigger.cpp

```diff
@@ -96,13 +137,415 @@ void AreaTrigger::Update(uint32 p_time)
         Remove(); // expired
 
     WorldObject::Update(p_time);
+    _timeSinceCreated += p_time;
+
+    if (GetTemplate()->HasFlag(AREATRIGGER_FLAG_HAS_ATTACHED))
+    {
+        if (Unit* target = GetTarget())
+            GetMap()->AreaTriggerRelocation(this, target->GetPositionX(), target->GetPositionY(), target->GetPositionZ(), target->GetOrientation());
```

Comment:
Well, I will go and sniff, would that help? But the triggers is moved with the caster. Also, it is not always created client-side, which is really weird. I tried sniffin Windwalking, but there was absolutely no smsg_update_object related to it. I got an aura update (157411 -> aura areatrigger), and immediately after it, i get the aura update for the movespeed buff spell, nothing between them, after them related to areatrigger ><